generator/routing_generator.cpp

```diff
@@ -413,19 +413,23 @@ void BuildRoutingIndex(string const & baseDir, string const & countryName, strin
               "Multiple:", multiple, "Equal:", equal));
 
   LOG(LINFO, ("Collect all data into one file..."));
-  string const fPath = localFile.GetPath(MapOptions::CarRouting);
 
-  FilesContainerW routingCont(fPath /*, FileWriter::OP_APPEND*/);
+  string const mwmPath = localFile.GetPath(MapOptions::Map);
+  string const fPath = mwmPath + ROUTING_FILE_EXTENSION;
```

Comment:
What does fPath mean? Could you please give it a more readable name? Why not routingPath?